tests/interu.yaml

```
@@ -0,0 +1,24 @@
+runners:
```

Todo in a new templating run

```suggestion
---
runners:
```

---

Good point, but this file will is not and will not be templated.

---

Na, as I mentioned, it can go into the next templating run.